source/modules/Git.rst

```diff
@@ -92,22 +92,22 @@ With git, you work with *local* repositories and *remotes* that they are connect
       origin  https://github.com/UWPCE-PythonCert/IntroPython-2017.git (push)
 
    This shows that the local repo on my machine *originated* from the one in
-   the UWPCE-PythonCert gitHub account (it shows up twice, because there is
-   a shortcut for both fetch from and push to this remote)
+   the UWPCE-PythonCert GitHub account (it shows up twice, because there is
+   a shortcut for both fetch from and push to this remote).
 
 .. rst-class:: build
```

Comment:
These directives are left over from when we were using this to make slides with Hieroglyph.

We should just remove them as we find them now.